Tests/src/PublicAPITests/MaterialTests.cpp

```diff
@@ -0,0 +1,315 @@
+/*
+ * Copyright 2023 Magnopus LLC
```

Comment:
[nit] 2024

Reply:
I believe this is something we can updated automatically

Reply:
As in it will get updated automatically, or we could in theory update it automatically? Seems odd to intentionally commit the wrong year. Not blocking the PR on this though.

Reply:
I think there is a script for this. I will speak to @MAG-AdamThorn again